### PR TITLE
Create changes-ee-5.3.md

### PR DESCRIPTION
**Pull request by yanzhiemq**

### CODE REVIEW

**Review by id — DISMISSED** [review]

File: en_US/changes/changes-ee-5.3.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Releases
+
+## e5.3.0
+## Enhancements
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion

### Enhancements
```

File: en_US/changes/changes-ee-5.3.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  This role could access all resources.
+  - viewer
+
+  This role can only view resources and data, corresponding to all GET requests in the REST API.
+
+- [#11612](https://github.com/emqx/emqx/pull/11612) During node evacuation, evacuate all disconnected sessions, not only those started with `clean_start` set to `false`.
+
+
+
+## Bug Fixes
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
### Bug Fixes
```

**Comment by JimMoen** [comment]
for e5.3.0 feature:

```
- [#11631](https://github.com/emqx/emqx/pull/11631) Add SSO feature and integrate with LDAP.

- [#11656](https://github.com/emqx/emqx/pull/11656) Integrated the SAML 2.0 Support for SSO.
```

**Comment by yanzhiemq** [comment]
> for e5.3.0 feature:
> 
> ```
> - [#11631](https://github.com/emqx/emqx/pull/11631) Add SSO feature and integrate with LDAP.
> 
> - [#11656](https://github.com/emqx/emqx/pull/11656) Integrated the SAML 2.0 Support for SSO.
> ```

@id I will add them in this repo directly, and @JimMoen will update them to emqx/emqx repo in next bugfix PR

File: en_US/changes/changes-ee-5.3.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  In this version, there are two predefined roles:
+  - superuser
+
+  This role could access all resources.
+  - viewer
+
+  This role can only view resources and data, corresponding to all GET requests in the REST API.
+
+- [#11612](https://github.com/emqx/emqx/pull/11612) During node evacuation, evacuate all disconnected sessions, not only those started with `clean_start` set to `false`.
+
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md:25]:
```suggestion
- [#11631](https://github.com/emqx/emqx/pull/11631) Add SSO feature and integrate with LDAP.

- [#11656](https://github.com/emqx/emqx/pull/11656) Integrated the SAML 2.0 Support for SSO.
```

File: en_US/changes/changes-ee-5.3.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  In this version, there are two predefined roles:
+  - superuser
+
+  This role could access all resources.
+  - viewer
+
+  This role can only view resources and data, corresponding to all GET requests in the REST API.
+
+- [#11612](https://github.com/emqx/emqx/pull/11612) During node evacuation, evacuate all disconnected sessions, not only those started with `clean_start` set to `false`.
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
```

File: en_US/changes/changes-ee-5.3.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+- [#11597](https://github.com/emqx/emqx/pull/11597) - Upgrade ekka to 0.15.13, which includes the following changes:
+  - upgrade Mria to 0.6.2:
+  - make bootstrap batch size configurable, [Mria PR](https://github.com/emqx/mria/pull/159)
+  - improve robustness of mria_membership processes, [Mria PR](https://github.com/emqx/mria/pull/156)
+  - fix log message formatting error
+  - Add `node.default_bootstrap_batch_size` option to EMQX configuration.
+  Increasing the value of this  option can greatly reduce a replicant node startup time, especially when EMQX cluster interconnect network latency is high and EMQX built-in database holds large amount of data, e.g. when the number of subscriptions is high.
+
+- [#11620](https://github.com/emqx/emqx/pull/11620) Add a new rule-engine SQL function `bytesize` to get the size of a byte-string. e.g. ``SELECT * FROM "t/#" WHERE bytesize(payload) > 10`
```
Comment (cherryxkz) [inline en_US/changes/changes-ee-5.3.md:17]: string. e.g. `SELECT * FROM "t/#" WHERE bytesize(payload) > 10`

File: en_US/changes/changes-ee-5.3.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  - Upgraded Mria to 0.6.2.
+  - Introduced the ability to configure the bootstrap batch size, as detailed in [Mria PR](https://github.com/emqx/mria/pull/159).
+  - Enhanced the reliability of mria_membership processes, as described in [Mria PR](https://github.com/emqx/mria/pull/156).
+  - Fix log message formatting error.
+  - Added `node.default_bootstrap_batch_size` option to EMQX configuration.
+  Increasing the value of this option can greatly reduce a replicant node startup time, especially when the EMQX cluster interconnect network latency is high and the EMQX built-in database holds a large amount of data, e.g. when the number of subscriptions is high.
+- [#11620](https://github.com/emqx/emqx/pull/11620) Added a new rule-engine SQL function `bytesize` to get the size of a byte-string. e.g. ``SELECT * FROM "t/#" WHERE bytesize(payload) > 10`.
+- [#11642](https://github.com/emqx/emqx/pull/11642) Updated to quicer version 0.0.200 in preparation for enabling openssl3 support for QUIC transport.
+
+- [#11608](https://github.com/emqx/emqx/pull/11608) Integrated the LDAP bind operation as a new authenticator.
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  - 可以通过配置设置 bootstrap 批量大小，[Mria PR](https://github.com/emqx/mria/pull/159)
+  - 提升了 mria_membership 进程的健壮性，[Mria PR](https://github.com/emqx/mria/pull/156)。
+  - 修复日志消息格式错误。
+  - EMQX 配置中添加了 `node.default_bootstrap_batch_size` 选项。 增加此选项的值可以极大地减少复制节点的启动时间，特别是当 EMQX 集群互连网络延迟较高且 EMQX 内置数据库包含大量数据时，例如订阅数较多的情况。
+
+- [#11620](https://github.com/emqx/emqx/pull/11620)  添加一个新的规则引擎 SQL 函数 `bytesize` 以获取字节字符串的大小。例如：``SELECT * FROM "t/#" WHERE bytesize(payload) > 10`。
+
+- [#11642](https://github.com/emqx/emqx/pull/11642) 将 quicer 升级到版本 0.0.200，为启用 OpenSSL3 对 QUIC 传输的支持做准备。
+
+- [#11608](https://github.com/emqx/emqx/pull/11608) 将 LDAP 绑定操作集成为新的身份验证器。
```
Comment (yanzhiemq) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
```

File: en_US/changes/changes-ee-5.3.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
+# Releases
+
+## e5.3.0
+
+### Enhancements
+
+- [#11568](https://github.com/emqx/emqx/pull/11568) Added support for defining templates for MQTT publish properties in the Republish rule action.
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
```

File: en_US/changes/changes-ee-5.3.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+
+- [#11610](https://github.com/emqx/emqx/pull/11610) Implemented a preliminary Role-Based Access Control for the Dashboard.
+
+  In this version, there are two predefined roles:
+  - Administrator: This role could access all resources.
+  
+  - Viewer: This role can only view resources and data, corresponding to all GET requests in the REST API.
+  
+- [#11612](https://github.com/emqx/emqx/pull/11612) During node evacuation, evacuate all disconnected sessions during node evacuation, not only those starting with `clean_start` set to `false`.
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+
+
+- [#11610](https://github.com/emqx/emqx/pull/11610) 在 Dashboard 中实施了初步基于角色的访问控制。
+
+  在此版本中，有两个预定义的角色：
+
+  - 管理员：此角色可以访问所有资源。
+  - 查看者：此角色只能查看资源和数据，对应于 REST API 中的所有 GET 请求。
+
+- [#11612](https://github.com/emqx/emqx/pull/11612)  在节点疏散期间疏散所有断开的会话，不仅仅是那些使用 `clean_start` 设置为 `false` 开始的会话。
```
Comment (yanzhiemq) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+- [#11631](https://github.com/emqx/emqx/pull/11631) 添加了单点登录（SSO）功能并与 LDAP 集成。
+- [#11656](https://github.com/emqx/emqx/pull/11656) 集成了 SAML 2.0 支持以实现单点登录（SSO）。
+
+### 修复
+
+- [#1111682](https://github.com/emqx/emqx/pull/1111682) 修复了在文件日志处理程序上将“旋转大小”设置为`infinity`时日志记录停止的问题。
```
Comment (yanzhiemq) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11682](https://github.com/emqx/emqx/pull/11682) 修复了在文件日志处理程序上将“旋转大小”设置为`infinity`时日志记录停止的问题。
```

File: en_US/changes/changes-ee-5.3.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+- [#11597](https://github.com/emqx/emqx/pull/11597) Upgraded ekka to 0.15.13, which incorporates the following changes:
+  - Upgraded Mria to 0.6.2.
+  - Introduced the ability to configure the bootstrap batch size, as detailed in [Mria PR](https://github.com/emqx/mria/pull/159).
+  - Enhanced the reliability of mria_membership processes, as described in [Mria PR](https://github.com/emqx/mria/pull/156).
+  - Fix log message formatting error.
+  - Added `node.default_bootstrap_batch_size` option to EMQX configuration.
+  Increasing the value of this option can greatly reduce a replicant node startup time, especially when the EMQX cluster interconnect network latency is high and the EMQX built-in database holds a large amount of data, e.g. when the number of subscriptions is high.
+- [#11620](https://github.com/emqx/emqx/pull/11620) Added a new rule-engine SQL function `bytesize` to get the size of a byte-string. e.g. ``SELECT * FROM "t/#" WHERE bytesize(payload) > 10`.
+- [#11642](https://github.com/emqx/emqx/pull/11642) Updated to quicer version 0.0.200 in preparation for enabling openssl3 support for QUIC transport.
+
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md:16]:
```suggestion
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+  - 可以通过配置设置 bootstrap 批量大小，[Mria PR](https://github.com/emqx/mria/pull/159)
+  - 提升了 mria_membership 进程的健壮性，[Mria PR](https://github.com/emqx/mria/pull/156)。
+  - 修复日志消息格式错误。
+  - EMQX 配置中添加了 `node.default_bootstrap_batch_size` 选项。 增加此选项的值可以极大地减少复制节点的启动时间，特别是当 EMQX 集群互连网络延迟较高且 EMQX 内置数据库包含大量数据时，例如订阅数较多的情况。
+
+- [#11620](https://github.com/emqx/emqx/pull/11620)  添加一个新的规则引擎 SQL 函数 `bytesize` 以获取字节字符串的大小。例如：``SELECT * FROM "t/#" WHERE bytesize(payload) > 10`。
+
+- [#11642](https://github.com/emqx/emqx/pull/11642) 将 quicer 升级到版本 0.0.200，为启用 OpenSSL3 对 QUIC 传输的支持做准备。
+
+
```
Comment (yanzhiemq) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
```

File: zh_CN/changes/changes-ee-5.2.md [hunk 14]
```diff
@@ -1,5 +1,32 @@
 # 版本发布
 
+## e5.2.1
+
+### 增强
+
+- [#11487](https://github.com/emqx/emqx/pull/11487) 将 bcrypt 的工作因子 (work factor) 限制在5-10的范围内，因为较高的值会消耗太多 CPU 资源。Bcrypt 库已更新以允许并行哈希计算。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.2.md]: 明确改动的是哪个模块。
```suggestion
- [#11487](https://github.com/emqx/emqx/pull/11487) 将认证功能中，基于 bcrypt 算法的密码加密的计算强度因子 (work factor) 限制在5-10的范围内，因为较高的值会消耗太多 CPU 资源。Bcrypt 库已更新以允许并行哈希计算。
```

File: zh_CN/changes/changes-ee-5.2.md [hunk 15]
```diff
@@ -1,5 +1,32 @@
...
+- [#11568](https://github.com/emqx/emqx/pull/11568) 在消息重发布规则动作中，支持设置 MQTT 5.0 发布属性与用户属性。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.2.md]:
```suggestion
- [#11568](https://github.com/emqx/emqx/pull/11568) 在消息重发布规则动作中，支持设置 MQTT 5.0 发布属性与用户属性。目前配置接口暂未完全集成到 Dashboard，将在后续版本中提供支持。
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+- [#11597](https://github.com/emqx/emqx/pull/11597)  将 Ekka 升级到 0.15.13，包括以下增强：
+
+  - 升级 Mria 到 0.6.2。
+  - 可以通过配置设置初始化阶段数据同步批量大小，[Mria PR](https://github.com/emqx/mria/pull/159)。
+  - 提升了 mria_membership 进程的健壮性，[Mria PR](https://github.com/emqx/mria/pull/156)。
+  - 修复日志消息格式错误。
+  - EMQX 配置中添加了 `node.default_bootstrap_batch_size` 选项。 增加此选项的值可以极大地减少复制节点的启动时间，特别是当 EMQX 集群互连网络延迟较高且 EMQX 内置数据库包含大量数据时，例如订阅数较多的情况。
+
+- [#11620](https://github.com/emqx/emqx/pull/11620)  添加一个新的规则引擎 SQL 函数 `bytesize` 以获取字节字符串的大小。例如：``SELECT * FROM "t/#" WHERE bytesize(payload) > 10`。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11620](https://github.com/emqx/emqx/pull/11620)  添加一个新的规则引擎 SQL 函数 `bytesize` 以获取字节字符串的大小。例如：`SELECT * FROM "t/#" WHERE bytesize(payload) > 10`。
```

File: en_US/changes/changes-ee-5.3.md [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+### Enhancements
+
+- [#11597](https://github.com/emqx/emqx/pull/11597) Upgraded ekka to 0.15.13, which incorporates the following changes:
+  - Upgraded Mria to 0.6.2.
+  - Introduced the ability to configure the bootstrap data sync batch size, as detailed in [Mria PR](https://github.com/emqx/mria/pull/159).
+  - Enhanced the reliability of mria_membership processes, as described in [Mria PR](https://github.com/emqx/mria/pull/156).
+  - Fix log message formatting error.
+  - Added `node.default_bootstrap_batch_size` option to EMQX configuration.
+  Increasing the value of this option can greatly reduce a replicant node startup time, especially when the EMQX cluster interconnect network latency is high and the EMQX built-in database holds a large amount of data, e.g. when the number of subscriptions is high.
+- [#11620](https://github.com/emqx/emqx/pull/11620) Added a new rule-engine SQL function `bytesize` to get the size of a byte-string. e.g. ``SELECT * FROM "t/#" WHERE bytesize(payload) > 10`.
```
Comment (wivwiv) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
- [#11620](https://github.com/emqx/emqx/pull/11620) Added a new rule-engine SQL function `bytesize` to get the size of a byte-string. e.g. `SELECT * FROM "t/#" WHERE bytesize(payload) > 10`.
```

Comment (wivwiv) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
- [#11620](https://github.com/emqx/emqx/pull/11620) Added a new rule-engine SQL function `bytesize` to get the size of a byte-string. e.g. `SELECT * FROM "t/#" WHERE bytesize(payload) > 10`.
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: en_US/changes/changes-ee-5.3.md (hunk 17 above)
Comment (yanzhiemq) [reply en_US/changes/changes-ee-5.3.md]: duplicated suggestion

File: en_US/changes/changes-ee-5.3.md [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+- [#11610](https://github.com/emqx/emqx/pull/11610) Implemented a preliminary Role-Based Access Control for the Dashboard.
+
+  In this version, there are two predefined roles:
+  - Administrator: This role could access all resources.
+  
+  - Viewer: This role can only view resources and data, corresponding to all GET requests in the REST API.
+  
+- [#11631](https://github.com/emqx/emqx/pull/11631) Added Single Sign-On (SSO) feature and integrated with LDAP.
+
+- [#11656](https://github.com/emqx/emqx/pull/11656) Integrated the SAML 2.0 Support for SSO.
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md:26]:
```suggestion
- [#11656](https://github.com/emqx/emqx/pull/11656) Integrated the SAML 2.0 Support for SSO.
   Note: AuthNRequest encryption is not supported yet.
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 19]
```diff
@@ -0,0 +1,57 @@
...
+
+- [#11610](https://github.com/emqx/emqx/pull/11610) 在 Dashboard 中实施了初步基于角色的访问控制。
+
+  在此版本中，有两个预定义的角色：
+
+  - 管理员：此角色可以访问所有资源。
+  - 查看者：此角色只能查看资源和数据，对应于 REST API 中的所有 GET 请求。
+
+- [#11631](https://github.com/emqx/emqx/pull/11631) 添加了单点登录（SSO）功能并与 LDAP 集成。
+- [#11656](https://github.com/emqx/emqx/pull/11656) 集成了 SAML 2.0 支持以实现单点登录（SSO）。
```
Comment (yanzhiemq) [inline zh_CN/changes/changes-ee-5.3.md:27]:
```suggestion
- [#11656](https://github.com/emqx/emqx/pull/11656) 集成了 SAML 2.0 支持以实现单点登录（SSO）。
   注意：暂时还不支持对AuthNRequest进行加密。
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 20]
```diff
@@ -0,0 +1,58 @@
...
+   注意：暂时还不支持对AuthNRequest进行加密。
```
Comment (yanzhiemq) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
```

File: en_US/changes/changes-ee-5.3.md [hunk 21]
```diff
@@ -0,0 +1,48 @@
...
+   Note: AuthNRequest encryption is not supported yet.
```
Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
```

**Review by wivwiv — APPROVED** [review]

**Review by Meggielqk — APPROVED** [review]

**Pull request merged**